_posts/2024-02-07-#28950.md

```diff
@@ -8,17 +8,43 @@ authors: [instagibbs]
 components: ["rpc"]
 host: ismaelsadeeq
 status: upcoming
-commit:
+commit: bf6fbea9794db5e8d0adce8ff6182574808cea54
 ---
 
-_Notes and questions to follow soon!_
-
-<!-- TODO: Before meeting, add notes and questions
 ## Notes
+- The `submitpackage` RPC receives topologically sorted transactions, a child, and its unconfirmed parents. It validates them against consensus and policy rules, if they passes these checks, the transactions are added to the mempool.
+
+- Packages submitted to the `submitpackage` RPC are only added to the node's mempool and not relayed to peers. Package relay is currently a work in progress.
+
+- `maxfeerate` is an optional parameter for the RPCs `sendrawtransaction` and `testmempoolaccept`. Users will use it to ensure that the transaction they are adding to their mempools and broadcasting to the network does not exceed some feerate, preventing accidental payment of an unreasonable transaction fee rate.
+
+- `maxburnamount` is also an optional parameter of `sendrawtransaction` that prevents broadcasting transactions with a data carrier `OP_RETURN` output that exceeds a certain value. The default value of `maxburnamount` is 0.
```

Comment:
```suggestion
- `maxburnamount` is an optional parameter of `sendrawtransaction` that prevents broadcasting transactions with a data carrier `OP_RETURN` output amount that exceeds a certain value. The default value of `maxburnamount` is 0.
```

Also: keeping sentence constructions identical between both parameters (this and the previous one) makes it easier for readers, e.g. : "... is an optional parameter **for** the ... **RPCs**". We like small diffs.

Reply:
Fixed